core/src/main/java/org/quickperf/annotation/DebugQuickPerf.java

```diff
@@ -16,6 +16,63 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * The <code>DebugQuickPerf</code> annotation displays both options from the JVM executing the test method (if any) or
```

Comment:
I know I had a hard time figuring out what and how to use this annotation (I checked how it was implemented to really understand it).
So, shouldn't we mention the Recorder stack ? Or is it sufficient since present in the output example ?

Reply:
> So, shouldn't we mention the Recorder stack ?

I don't think. Other types of information may be displayed later with @DebugQuickPerf. I don't want to have each time to update the description.

> Or is it sufficient since present in the output example ?

Yes, I think.